hw3_bem_graphics_design/responsive_layout/app/index.html

```diff
@@ -0,0 +1,320 @@
+									<a class="sidebar__category-title" href="#"> File Manager </a>
+								</div>
+								<a class="sidebar__category-arrow" href="#">
+									<img class="sidebar__arrow-icon" src="images/arrow-down.svg" alt="arrow icon" />
+								</a>
+							</li>
+						</ul>
+					</div>
+				</div>
+				<div class="sidebar__icon-bar">
+					<button class="sidebar__icon-bar-btn">
+						<img class="sidebar__icon-bar-logo" src="images/Settings.svg" alt="logo" />
+					</button>
+					<button class="sidebar__icon-bar-btn">
+						<img class="sidebar__icon-bar-logo" src="images/logout.svg" alt="logo" />
+					</button>
+					<button class="sidebar__icon-bar-btn">
+						<img class="icon-bar-logo" src="images/globe.svg" alt="logo" />
+					</button>
+				</div>
+			</aside>
+
+			<div class="main-area">
+				<div class="header__container">
+					<div class="header__additional-line">
+						<a class="header__additional-line-logo" href="#">
+							<img src="images/Logo.svg" alt="company logo" />
+						</a>
+						<a class="header__additional-line-avatar" href="#">
+							<img src="images/photo-person.svg" alt="photo" />
+						</a>
+					</div>
+					<header class="header">
+						<form class="header__search">
+							<button class="header__search-icon">
+								<img src="images/search-icon.svg" alt="icon" />
+							</button>
+							<input class="header__search-input" type="text" placeholder="Search..." />
+						</form>
+						<div class="header__icon-bar">
+							<button class="header__icon-bar-btn">
+								<img class="header__icon-bar-logo" src="images/Notification.svg" alt="logo" />
+							</button>
+							<button class="header__icon-bar-btn">
+								<img src="images/calendar.svg" alt="logo" />
+							</button>
+							<button class="header__icon-bar-btn">
+								<img src="images/different.svg" alt="logo" />
+							</button>
+							<a class="header__icon-bar-avatar" href="#">
+								<img
+									class="header__icon-bar-avatar-photo"
+									src="images/photo-person.svg"
+									alt="photo"
+								/>
+							</a>
+						</div>
+					</header>
+				</div>
+
+				<div class="premium">
+					<div class="premium__content">
+						<h1 class="premium__content-title">Unlock premium stats</h1>
+						<h2 class="premium__content-text">Get up to 10TB of storage for a limited time</h2>
+					</div>
+					<button class="premium__btn">
+						<img class="premium__btn-image" src="images/lightning.svg" alt="image" />
+						<span class="premium__btn-text">Upgrade</span>
+					</button>
+				</div>
+
+				<div class="cards">
+					<div class="cards__item">
+						<p class="cards__item-title">Revenue</p>
+						<p class="cards__item-digit">$56,945</p>
+						<div class="cards__item-bottom-bar">
+							<div class="cards__item-condition-container cards__item-condition-container--first">
+								+45%
+							</div>
+							<p class="cards__item-text">From 4.6%</p>
+						</div>
+					</div>
+					<div class="cards__item">
```

Comment:
![Screenshot 2023-03-31 at 14 11 06](https://user-images.githubusercontent.com/70685561/229092698-d8390253-e3bd-4756-81ed-0c054089b252.png)
It has full width in design